//---------------------------------------------------------------------
//---------------------------------------------------------------------
syntax = "proto3";

//---------------------------------------------------------------------
//---------------------------------------------------------------------
package ni.measurementlink.measurement.tests.v1;

//---------------------------------------------------------------------
//---------------------------------------------------------------------
option csharp_namespace = "NationalInstruments.MeasurementServices.Measurements";

//---------------------------------------------------------------------
//---------------------------------------------------------------------

message SampleMeasurementParameter{
    float float_in=1;
    repeated double double_array_in=2;
    bool bool_in=3;
    string string_in =4;
    repeated string string_array_in =5;
}
